typing/ctype.ml

```diff
@@ -1971,32 +1971,53 @@ let enter_poly env univar_pairs t1 tl1 t2 tl2 f =
 
 let univar_pairs = ref []
 
-(* assumption: [ty] is fully generalized. *)
-let reify_univars ty =
-  let rec subst_univar scope vars ty =
+(**** Instantiate a generic type into a poly type ***)
+
+let polyfy env ty vars =
```

Comment:
Sorry to bother a native speaker, but why is this not `polify`?

Reply:
Well, the name isn't a real word so the spelling is open to debate. The function makes a type into a `Tpoly` type hence it "poly"-fies it. The name is also a pun on `unify`: poly- is the greek prefix for many and uni- the latin prefix for one.

`polify` is probably more consistent with actual "-fy" words, but I think polyfy makes the meaning clearer.

Reply:
I find it confusing to see a name that I can't really pronounce (well in that case, I can pronounce it but it sounds like a spelling mistake).